sphinx/source/docs/user_guide/geo.rst

```diff
@@ -1,59 +1,54 @@
 .. _userguide_geo:
 
-Mapping Geo Data
+Mapping geo data
 ================
 
-Bokeh has started adding support for working with Geographical data. There are
-a number of powerful features already available, but we still have more to add.
-Please tell us your use cases through `Discourse`_ or on `GitHub`_ so that we
-can continue to extend these features to meet your needs.
+Bokeh has started adding support for working with geographical data. There are
+a number of powerful features already available. Tell us your use cases through `Discourse`_ or on `GitHub`_ so that we can continue to extend these features to meet your needs.
```

Comment:
As long as we are editing this file, I think this old intro para can just be deleted. We are past the point of "starting" and so this is just extra distraction.

Reply:
This page will still need some sort of intro, though. How about something short along the lines of:

> Bokeh supports creating map-based visualizations and working with geographical data.